internal/distro/fedora/package_sets.go

```diff
@@ -422,13 +422,7 @@ func anacondaPackageSet(t *imageType) rpmmd.PackageSet {
 		},
 	})
 
-	releasever := t.Arch().Distro().Releasever()
-	version, err := strconv.Atoi(releasever)
-	if err != nil {
-		panic("cannot convert releasever to int: " + err.Error())
-	}
-
-	if version <= 36 {
+	if common.VersionLessThan(t.arch.distro.osVersion, "37") {
 		ps.Append(rpmmd.PackageSet{
```

Comment:
Oh my, this should be

```suggestion
		ps = ps.Append(rpmmd.PackageSet{
```

Currently, it's a no-op, since `ps` is passed by value into `Append`. :facepalm:

Reply:
D:, fixed.